core/src/components/range/range.md.scss

```diff
@@ -54,7 +59,7 @@
 }
 
 :host(.range-has-pin) {
-  @include padding($range-md-padding-vertical + $range-md-pin-font-size + $range-md-pin-padding-vertical, null, null, null);
+  @include padding($range-md-pin-dimension, null, null, null);
```

Comment:
The existing padding did not account for the new pin size which is why I changed it here